src/main/resources/db/migration/V1.1_001__message_received_precision.sql

```diff
@@ -0,0 +1,4 @@
+
+-- github issue 8 / add millisecond resolution
+alter table message modify received_date timestamp(3) not null;
```

Comment:
Do we perhaps need to use db-specific flyway locations? https://flywaydb.org/documentation/faq#db-specific-sql